include/tvm/tir/schedule/schedule.h

```diff
@@ -601,9 +601,11 @@ class ScheduleNode : public runtime::Object {
    * \param buffer_index The index of the buffer in block's read or write region.
    * \param buffer_index_type The type of the buffer index, kRead or kWrite.
    * \param index_map The transformation to apply.
+   * \param pad_value The value to write into padding introduced by the transformation.
    */
   virtual void TransformLayout(const BlockRV& block_rv, int buffer_index,
-                               BufferIndexType buffer_index_type, const IndexMap& index_map) = 0;
+                               BufferIndexType buffer_index_type, const IndexMap& index_map,
+                               const Optional<PrimExpr>& pad_value) = 0;
```

Comment:
How about adding a default value? 
```suggestion
                               const Optional<PrimExpr>& pad_value = NullOpt) = 0;
```

Reply:
Thank you for the catch, and updated.  This also makes the C++ API closer to the Python API.